frontend/src/app/[locale]/search/error.tsx

```diff
@@ -91,10 +91,6 @@ export default function SearchError({ error, reset }: ErrorProps) {
   const { agency, category, eligibility, fundingInstrument, query, status } =
     convertedSearchParams;
 
-  useEffect(() => {
-    console.error(error);
-  }, [error]);
-
```

Comment:
not sure how this came back, and the above change I think I also handled in a previous PR 🤷